ecosystem/indexer/src/indexer/tailer.rs

```diff
@@ -154,7 +188,7 @@ impl Tailer {
     ) -> anyhow::Result<Vec<Result<ProcessingResult, TransactionProcessingError>>> {
         let mut tasks = vec![];
         for processor in &self.processors {
-            let txn2 = txn.clone();
+            let txn2 = remove_null_bytes_from_txn(txn.clone());
```

Comment:
this gets repeated on every clone, is there a way to do this above the `for processor in &self.processors`?